exercises/practice/freelancer-rates/.meta/config.json

```
@@ -0,0 +1,9 @@
+{
+  "blurb": "Using Floats and Integers, help a freelancer calculate their rates.",
+  "authors": [],
+  "files": {
+    "solution": ["src/freelancer_rates.gleam"],
+    "test": ["test/freelancer_rates_test.gleam"],
+    "example": ["./example.gleam"]
```

```suggestion
    "example": [".meta/example.gleam"]
```

It's relative to the solution directory root!

---

(You marked this as resolved but still there!)